dxr/plugins/python/indexers.py

```diff
@@ -194,12 +187,23 @@ def visit_ClassDef(self, node):
                               name=name, qualname=qualname,
                               start=start, end=end)
 
+        # Show a menu when hovering over this class.
+        self.yield_ref(start, end,
+                       menus.class_menu(self.file_to_index.tree, class_name))
```

Comment:
It seems like either `class_menu` or `menus.class` (ignoring the keyword for now) would be sufficient to expose your intent. I'd recommend either importing deeper (my favorite) or shortening the routine names.